content/03.results.md

```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
-We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.
-We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures [@doi:10.1038/s43018-020-0027-5] (RefSig) across cancer groups **Figure {@fig:Fig3}E** and samples **Figure {@fig:S4}A**.
-Stage 0 and/or 1 tumors characterized by low TMBs **Figure {@fig:S2}G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
-Signature N6 is CNS-specific signature which we observe nearly universally across samples.
-Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
-Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
-Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure {@fig:S4}B**.
-This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.
+**TODO: A lot of different denominators here with no clear meaning. What is 666 vs 872 vs 54?**
+
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). **TODO: Clarify more what "top 50" means. Most mutations? Most commonly mutated across samples? etc.**
+_TP53_ was the most frequently mutated gene across OpenPBTA tumors (8.4%, 56/666), significantly co-occurring with _H3F3A_ (OR = 32, 95% CI: 15.3 - 66.7, q = 8.46e-17), _ATRX_ (OR = 20, 95% CI: 8.4 - 47.7, q = 4.43e-8), _NF1_ (OR = 8.62, 95% CI: 3.7 - 20.2, q = 5.45e-5), and _EGFR_ (OR = 18.2, 95% CI: 5 - 66.5, q = 1.6e-4). 
+Other canonical cancer driver genes that were frequently mutated included _BRAF_, _H3F3A_, _CTNNB1_, _NF1_, _ATRX_, _FGFR1_, and _PIK3CA_. 
+Although LGG and embryonal tumors made up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stemmed from HGGs (N = 847/872). **TODO: Is this denominator the total number of significant? Unclear what these numbers are**
```

Comment:
woops ,put this TODO in twice! focus on the one at the top of this section, line 158.

Reply:
Now I am second guessing myself:
- 666 is the total N of tumors in the co-occurrence module (across all PBTA) from sum of Ns in rows `cooccur_top50.ALL.tsv`
- Seems like `847/872` comes from:
```r
co <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/interaction-plots/results/cooccur_top50.ALL.tsv") %>%
  filter(q < 0.05)
```
This has nrow = 872 significant events
```r
hgg <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/interaction-plots/results/cooccur_top50.Diffuse-astrocytic-and-oligodendroglial-tumor.tsv") %>%
  filter(q < 0.05)
```
This has nrow 847

Reply:
Ok, we should probably add a (likely extremely small!) notebook to the module then to calculate these numbers. They need to have a direct source in the repo.

Reply:
PR: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1398